healpy/sphtfunc.py

```diff
@@ -972,9 +979,9 @@ def pixwin(nside, pol=False):
     pw = pf.getdata(fname)
     pw_temp, pw_pol = pw.field(0), pw.field(1)
     if pol:
-        return pw_temp, pw_pol
+        return pw_temp[:lmax], pw_pol[:lmax]
     else:
-        return pw_temp
+        return pw_temp[:lmax]
```

Comment:
shouldn't this be `[:lmax+1]`?

Reply:
You're right - I made these two changes (3*nside instead of 3*nside-1, and [:lmax] instead of [:lmax+1]) to that they'd cancel out.

This doesn't work of `lmax` is actually provided, so let me push a fix.